framework/contrib/hit/parse.cc

```diff
@@ -183,6 +183,12 @@ Node::~Node()
   }
 }
 
+void
+Node::remove()
+{
+  delete this;
```

Comment:
😆

Reply:
A quirk of our inability to bridge the python --> c++ bindings gap w.r.t. memory management.